### ISSUE TITLE
[Bug]: Notifications incorrectly aligns text when iOS font is larger

### ISSUE THREAD

**Issue by noahtallen** [Automattic/wp-calypso#68702]
### Quick summary

I have iOS larger font sizes enabled to make text bigger on my phone. In this scenario, the notifications panel appears to incorrectly align text.

![71B9F7CA-3FC3-4E77-AC83-4550E02017D1](https://user-images.githubusercontent.com/6265975/194258889-17ec256f-45d6-4f32-9da3-9f5699655e01.png)

![3D14305E-E43E-4DDC-9E17-FD4754FC77AB](https://user-images.githubusercontent.com/6265975/194259054-38a68c3c-475a-423b-882f-4f043efe2860.jpeg)

### Steps to reproduce

Open the notifications panel on the mobile front-end (outside of calypso itself)

### What you expected to happen

Font should be correctly aligned

### What actually happened

Font is cut off

### Browser

Apple Safari

### Reproducibility

Consistent

**Comment by cuemarie** [comment]
_This report was found during a recent backlog sweep._

📌 SCRUBBING : RESULT - _Replicated / Could Not Replicate / Uncertain_
- Tested on Jetpack App - Could not replicate
- Tested on Mobile Safari Browser - Could not replicate

📌 FINDINGS/SCREENSHOTS/VIDEO
- The WordPress.com App no longer shows notifications, in favor of the Jetpack App, so I tested there as well as in the iOS mobile Safari browser.
- The Safari Browser window did not completely respond to the text size change, but did not cut words off.
- The Jetpack App responded as expected to the larger text.

| Accessibility Setting  | Jetpack App | Safari Browser |
| ------------- | ------------- | ------------- |
| ![IMG_8532](https://user-images.githubusercontent.com/27249804/236067062-0a7cff15-2cdc-45d4-ab8e-3e507cf135f6.PNG) | ![IMG_8531](https://user-images.githubusercontent.com/27249804/236067349-2a3c8e8c-5f4d-49e4-bfcb-aa7dda2b615c.PNG) | ![IMG_8530](https://user-images.githubusercontent.com/27249804/236066984-dc129512-9c6c-4ae9-8b73-189b703a103e.PNG) |

📌 ACTIONS
- Closed as not reproducible/no longer relevant
